extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -468,6 +470,13 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     }
   }
 
+  private ensureHasParentNode = (el: JQuery<HTMLElement>) => {
```

Comment:
@tomholub because it's a common DOM helper, should it be moved somewhere else from here?

Reply:
I can only imagine ever needing it in Gmail webmail, so I think it's placed correctly.